### PR TITLE
Use try_from_glib shortcut

### PR DESCRIPTION
**Pull request by fengalin**
See https://github.com/gtk-rs/gtk-rs-core/pull/3.

I also renamed the function specifiers for `RustTypeBuilder` that I introduced in https://github.com/gtk-rs/gir/pull/1092/commits/c43d9a5c8314631ccf7350e5750d7df96e49c625 because I thought they were uselessly verbose.

### CODE REVIEW

File: src/config/error.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ impl TomlHelper for toml::Value {
             None => return,
         };
         for (key, _) in table.iter() {
-            if options.iter().find(|&entry| entry == key).is_none() {
+            if options.iter().any(|&entry| entry == key) {
```
Comment (MarijnS95) [inline src/config/error.rs:24]: Aren't you missing a `!` here?

```suggestion
            if !options.iter().any(|&entry| entry == key) {
```

But slices also have a contains method:
```suggestion
            if !options.contains(&key.as_str()) {
````

**Review by MarijnS95 — CHANGES_REQUESTED** [review]

File: src/config/error.rs (hunk 1 above)
Comment (fengalin) [reply src/config/error.rs:24]: Ah right... I probably rushed the clippy pass and it didn't show up when I regen `gstreamer-rs`.

**Comment by fengalin** [comment]
It seems CI breakage is unrelated, I'll check this later this weekend or next week.

**Comment by sdroege** [comment]
> It seems CI breakage is unrelated, I'll check this later this weekend or next week.

Yes that's leftover work for @bilelmoussaoui and @GuillaumeGomez from splitting the repos. The CI here also needs to be updated.

**Review by sdroege — APPROVED** [review]

**Review by MarijnS95 — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Ah right, just let me fix it quickly and then we can restart the CI.

**Comment by GuillaumeGomez** [comment]
Opened  #1137.

**Comment by GuillaumeGomez** [comment]
Restarted CI.

**Comment by GuillaumeGomez** [comment]
Ah right, you need to rebase to have the "new" CI. ;)

**Comment by fengalin** [comment]
Thanks @GuillaumeGomez

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**